Spatial.gd

```diff
@@ -760,3 +861,39 @@ func _on_Settings_pressed():
 	settings_dialog.load_settings()
 	settings_dialog.show()
 
+
+func _on_MarkerPacks_cell_selected():
+	var category_item = marker_packs.get_selected()
+	self.currently_active_category = category_item.get_metadata(0)
+	for path in self.paths.get_children():
+		if path.category_name == find_pedigree_name(category_item, category_item.get_metadata(0).get_name()):
+			path.is_editable = true
+			self.currently_active_path = path
```

Comment:
I don't think you need to set currently_active_path here because this will just be overwritten until we reach the final path. If the user wants to edit a path they can just click on the path they want to edit to select it.